internal/rootcoord/root_coord.go

```diff
@@ -1263,15 +1263,23 @@ func (c *Core) describeCollectionImpl(ctx context.Context, in *milvuspb.Describe
 		allowUnavailable: allowUnavailable,
 	}
 
-	if err := c.scheduler.AddTask(t); err != nil {
-		log.Info("failed to enqueue request to describe collection", zap.Error(err))
-		metrics.RootCoordDDLReqCounter.WithLabelValues("DescribeCollection", metrics.FailLabel).Inc()
+	ts, err := c.tsoAllocator.GenerateTSO(1)
```

Comment:
we may not be able to do so.

This fix break the consistency of proxy collection cache.

Right now, all describe need to wait until other ddl done.